db/structure.sql

```diff
@@ -1,6 +1,7 @@
 SET statement_timeout = 0;
 SET lock_timeout = 0;
 SET idle_in_transaction_session_timeout = 0;
+SET transaction_timeout = 0;
```

Comment:
not sure that we need this line

Reply:
sorry)
It is my local settings.

Reply:
Thank you for attention :heart_decoration: